Libraries/D3DX12/d3dx12.h

```diff
@@ -1881,7 +1881,7 @@ inline UINT64 GetRequiredIntermediateSize(
     UINT64 RequiredSize = 0;
     
     ID3D12Device* pDevice = nullptr;
-    pDestinationResource->GetDevice(__uuidof(*pDevice), reinterpret_cast<void**>(&pDevice));
+    pDestinationResource->GetDevice(IID_ID3D12Device, reinterpret_cast<void**>(&pDevice));
```

Comment:
The main impact here is that you need to be linking against ``dxguid.lib``. Almost all DirectX apps are already doing this, so it shouldn't be a major issue.